src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3815,6 +3920,48 @@ components:
       default: gcp
       description: Cloud platforms supported by TDR.
 
+    ##############################################################################
+    ## SEARCH API STANDARD MODELS
+    ##############################################################################
+    SearchIndexRequest:
+      required:
+        - sqlQuery
+      type: object
+      properties:
+        sqlQuery:
+          type: string
+          description: >
+            SQL to index BigQuery columns into Elasticsearch
+    SearchIndexModel:
+      type: object
+      properties:
+        indexSummary:
+          type: string
+          description: Summary of this index
```

Comment:
Can you specify it's the es json and provide a link to the docs or an example?

Reply:
Ultimately, it isn't totally clear what the output will look like, so I hesitate to add anything beyond `Summary of this index from Elasticsearch` (which I changed below). There are also a few other issues. It's not possible to attach the `externalDocs` openapi annotation to individual properties. If I attach that annotation to the API definition instead, it will likely be misleading since our Java API will be somewhat different from the raw Elasticsearch API. Additionally, the JSON responses from both ES querying and indexing are really large, so adding those as examples isn't very informative.

Reply:
I think the inputs and outputs of the new endpoints will change a bit during the iterations leading up to the MVP release. Once things settle down we can spend the time to improve the openapi descriptions and schemas.